apps/src/lab2/types.ts

```diff
@@ -189,6 +190,15 @@ export interface VideoLevelData {
   thumbnail: string;
 }
 
+// Addtional fields for videos that are linked as references in the
+// Help & Tips tab of Instructions.
+interface VideoData extends VideoLevelData {
+  name?: string;
+  key?: string;
+  enable_fallback?: boolean;
+  autoplay?: boolean;
+}
```

Comment:
I'm following the data shape defined [here](https://github.com/code-dot-org/code-dot-org/blob/9c9f417105a49ac9d6cc71235229425bc44d4854/apps/src/templates/types.js#L3). 

Side note: it seems strange to me that this is in the only shape in such a generic file, but I'll leave that puzzle for another day.